sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -2344,8 +2344,11 @@ case class DateDiff(endDate: Expression, startDate: Expression)
     copy(endDate = newLeft, startDate = newRight)
 }
 
+// scalastyle:off line.size.limit
 @ExpressionDescription(
-  usage = "_FUNC_(year, month, day) - Create date from year, month and day fields.",
+  usage = """_FUNC_(year, month, day) - Create date from year, month and day fields.
+    If the configuration `spark.sql.ansi.enabled` is false, the function returns NULL on invalid inputs. Otherwise, it will throw an error instead.
```

Comment:
can you run `DESC FUNCTION` locally and see if the output is reasonable? IIRC we need to put a super long single-line string for the doc.

Reply:
ok, updated